examples/tutorials/pipeline_end_to_end_tutorial.ipynb

```diff
@@ -0,0 +1,1472 @@
azKsAJuunoEDcYK4xxjCmfsO7cdAiNs6nLmiHI545Lx6/HHH3/BWyGCltUsjLkOhQ/tKb16LgQKgUKgECgECoFCoBAoBAqBQqAQKAQKgZ1HoNQwO49xlVAIHAgEKA8ZReyl75B1W8DwX/ziF0eF5Ic+9KFxKxiKRlvPPNP2HrL//rIGEWD5utq5JRSats6i4Lz99tvHL8KtVFF+uUJgOxG4555h+OxnZ4ent52NxsPUt2IMmfJmOy4HtDuP5J/+aWgrrYamrB9a/5gdwj5NX8+LI2CMYWjICjLbaDmv44477hgNGh/96EfHlWzGFY7hwjgyNYgwVDB8MLLYLpCxl5EkWwNOOXKeCcOtcYohxEo5BuOvfOUr4+oQz1aY/OEf/uELhhCrWsoVAoVAIVAIFAKFQCFQCBQChUAhUAgUAoXA3iFQBpG9w75KLgReRMBn5E3pPx5g4DN1BxA0JVs7PGN2AEEzLIyHEHjeI4Ua5SBlHiUfxSElIMWiVSC2qvn+978/bnMl3lYxDBrXXHPNuNXMiSee+KJhxFfaPpMnp1OqHYhMW9wMLm3vm+H5ttLkyXaYu/NKKDBjiDnnnHOGd7zjHXO/1H4RyBW6I6c6feKJoS2lmclsqYCD6O2ZpE7bl+3tU/UVYvrwscLooZocnu7MECtDGCtUlWa5nU4319w1/X/7t2H4/OeH4X3vG9qZONtZyvbRouSn4LeywioI95yxQL/U/6228LyXTvl4ydhkRYexygo22/pZZebgdSvZhBuPGHCnhlXyoWVbK/K6P//888czjMRNz/v4YRu/bIdlVQrjCCOKLQStHHFOiK28jhw5Mq5QYbTZa5z2so6q7EKgECgECoFCoBAoBAqBQqAQKAQKgUJgVRAog8iq1ETxcXgRoHW15UrbWmXwmfrXvz7TynpuXz23vaiG4f3vH5qWbrbXTlNANs3anuJFsec8EVvR2Fvf6o1PfOITo0GEocSX2VaIUFJaVXJ8MwKMK0XIyhjitOm2t/6oFWYwIH8zpNhD6Pl3v3t4vOFwX/MMLQwuZ5111vDuFv4nf/In4xffeyr8IoVHTnsu0Xw7odtp3YxcjCEXXDCr04bhuERAfe5xnS4i1kFMo0qcE+Lw87awYDSGMJLslNPULVSwCuVf/mXW7FfVIMIIQLnPmGCllnt9n2FAv+T18VVZ9YA3nvHUIegMFVaL/FNbkmPFiHOJjEVWrjFYXND6ofQc4w/5+KyCE2fFG/kYPCKncOkZXIx7xjtnK+XAdGeOGK+Mj8orVwgUAoVAIVAIFAKFQCFQCBQChUAhUAgUAquDQBlEVqcuipPDhoCvrfnrrx+GG2+crSJ4+OGZZpYxhKY2K0Roa9uXxm0j+qF91jw7fICi7agyb6+gy178lI+2pbFVzL+2E6l9ie3L7K+2E6opTN/QDDuvaUaQY269dSaffYPaKpJx5YRP5uFghUiT/5ibbhrOaWG/bZrjU9oKk9+0fYXOatvOnN2UnPbzp6BcWUcO2vXvfnem8W4K2HGPJPIyBNGG45/hqxmRmlVptlKERly9qs89rtOVxXaHGGOPc5B6O65mNIYcPWZih0p7kazm0PT1o23QVlptAcHYBV5MsXd32aqOEcDqEFeGgqwQ0e+F28pO/35FG6diHImBYe+4n5WclSA5/8gzowXjDh4ZSrKCA/9k6eVFhdGDnAyz4q2KIafwB9tSohvbuP3QQw+N4528DCC27kLXtlvTLbn2GpMqvxAoBAqBQqAQKAQKgUKgECgECoFCoBAoBNpW2gVCIVAI7BECtKBWDXzjG7NPxW2rRJkuvCkfX3AU5O2sjnHLrLZSYtSgtv3vm/Zt9nn5Cwl3/4Zi0coPCkCKwHvbvkMOEaZQpUC9tRlAXtMMAB9uKz9ObRrg4z7zmZlRpG2vNWqfaYWbcvEF1ww/x7btuE5rW0m9vhl/3nbZZcNxbU+hUy69dHi5FSSr7BhDssrHYRQMXQxczz03q9Oed9pvK0fUYTMmtU/Pnd48O2F7lQ0+vQwH4F7Ts2ubrbI0SVtn7ZbLyhRGEb4dVTEesL5b5c8rh6LfVlH6LyOAK4MAQ4i43un7MYgwUjKSWA3B8LAKRhH82ELL+R/OBbFNFsNHZLu7LdFx4LotrhzKTt5sC9bLSn5jGZnIllUzd95557gyztaA0mRrvxhDrIwrVwgUAoVAIVAIFAKFQCFQCBQChUAhUAgUAquHwMv+d3Orx1ZxVAgcAgR8kv73fz8MX/va0PZcmSnTaWStIpg6yvb2BfILRgTpmnJvXDEyTbsHzxSJlKM/aGd/PNyMABSPFIQUi69rysJLW9xJzThyfDMSHBMjQQsfV4b0/FK6tvBjmyL2uGYsOb7JzKDyivPOG45ZdYMIDXdbITN88pPD8K1vzVaGNCXrWG/qr3fkZPRiEGIwgYV722nRjJfbcQRUiSpoeu2h2enal/6zrax2vOCjBaQJ0Ju3RQdNMT+zie1W+fPKofi3gsLqLtesCukNBNN84hgJjAEcAyljxKo4Kzds92Uli9UdjCRZ0cKQ85a2JSGeGTXIsoisjL5WhzAAo80I4qwRZ484l8SWXQzE5QqBQqAQKAQKgUKgECgECoFCoBAoBAqBQmD1EKgVIqtXJ8XRQUeA4pBxwzZKFOhOc37qqZlxgJZ0nqMwp72ldLdahMLRKpF3vWu2umAFFJC+oLY3v4PUKQx9cf1cWwHz5nZgwmscmOAzeCdWU/yTZ56DTfMva/Eva9dXkNW2UlbO0Biv6n786sZJ3O2cgnG5gS3OrBYhzzwXA5d8VgRZLdIUtk2rOtsazX3Ds9zOIaAJOsrGwqzHH99dYwipNA3NgH2wnfk9LhSya9peOYYARg2GEMbNGAjW4yd5YkSwgoLRgUFkFVaJ4J0B5G1ty72cHeJcEas6GEisDiGzVSPZDmwtecnIZ1utV7eVbG9961vHMQ8dW3MZ/5ytVKtD1kKxwguBQqAQKAQKgUKgECgECoFCo
```

Comment:
I personally would like to see a section containing links back to our internal docs 

https://docs.lineapy.org/en/latest/fundamentals/concepts.html#pipeline-concept
https://docs.lineapy.org/en/latest/guide/build_pipelines/index.html 

especially since so much of the content is taken from that.

Reply:
Yeah, I can add the section at the end of the notebook as a "what next" pointer